pkg/store/container/container.go

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package container
+
+import (
+	"sync"
+
+	"github.com/kubernetes-incubator/cri-containerd/pkg/store"
+)
+
+// Container contains all resources associated with the container. All fields of
+// Container are thread safe.
+type Container struct {
+	// Metadata is the metadata of the container, it is **read-only**.
+	Metadata
+	// Status stores the status of the container.
+	Status StatusStorage
+	// TODO(random-liu): Add containerd container client.
```

Comment:
I mean in the future, we could [this](https://github.com/containerd/containerd/blob/master/container.go#L22) here, after we start to use containerd client.